template/tsconfig/cypress-ct/tsconfig.cypress-ct.json

```diff
@@ -2,8 +2,8 @@
   "extends": "./tsconfig.app.json",
   "include": [
     "env.d.ts",
-    "src/**/*",
-    "src/**/*.vue",
+    "src/**/__tests__/**/*",
```

Comment:
By only including those under `__tests__` by default, we avoid type-checking most of the component files twice; only the components that are also `import`ed by Cypress Component Testing specs would be type-checked again in the `type-check:cypress-ct` task.